reconcile/utils/terrascript_client.py

```diff
@@ -173,6 +173,7 @@ def download_logtoes_zip(self, release_url):
         if not os.path.exists(zip_file):
             r = requests.get(zip_url)
             r.raise_for_status()
+            # pylint: disable=consider-using-with
```

Comment:
Can you file a task to handle these consider-using-with cases? It's an excellent first issue for a newcomer.

Reply:
https://issues.redhat.com/browse/APPSRE-3686